ProcessLib/SecondaryVariable.h

```diff
@@ -159,41 +163,39 @@ class SecondaryVariableCollection final
 
 //! Creates an object that computes a secondary variable via extrapolation
 //! of integration point values.
-template<typename PropertyEnum, typename LocalAssembler>
-SecondaryVariableFunctions
-makeExtrapolator(PropertyEnum const property,
-                 NumLib::Extrapolator<PropertyEnum, LocalAssembler>&
-                 extrapolator,
-                 typename NumLib::Extrapolator<PropertyEnum,
-                     LocalAssembler>::LocalAssemblers const& local_assemblers)
+template <typename LocalAssemblerCollection,
+          typename IntegrationPointValuesMethod>
```

Comment:
can you add documentation of `IntegrationPointValuesMethod`? I am not sure what the method should do.